tests/storage.test.js

```diff
@@ -12,7 +12,7 @@ describe('STORAGE', function () {
   });
 
   it('The storage class should be OK.', function (done) {
-    should.exist(require('../lib/storage'));
+    should(require('../lib/storage')).be.ok;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2759952634/issues/source?bid=2452374&fileBranchId=3360528#l15)